exercises/Cielo-Raymundo/short-response.md

```diff
@@ -0,0 +1,21 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+    Function parameters are "placeholders" that go in the function that have no value, but are being used. Arguments are actual values that go 
```

Comment:
Yes to the use of 'placeholders'. A more precise way of saying "go in the function" is that arguments are "passed into functions" and they take the place of their parameters.